manage_people/migrations/0001_mp_initial.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import  migrations, models, transaction
+
+MANAGE_PEOPLE_ROLE_DATA = [
+    # NOTE: this data does not need to vary between environments, as the ids
+    #       in the user_role table are the same in both qa and prod oracle
+    # (user_role_id, canvas_role_label, xid_allowed)
+    (0, 'StudentEnrollment', False),
+    (1, 'Course Head', False),
+    (2, 'Faculty', False),
+    (5, 'TaEnrollment', False),
+    (7, 'DesignerEnrollment', False),
+    (9, 'TeacherEnrollment', False),
+    (10, 'Guest', True),
+    (11, 'Course Support Staff', False),
+    (12, 'Teaching Staff', False),
+    (14, 'Shopper', False),
+    (15, 'ObserverEnrollment', False)
+]
+LTI_PERMISSIONS_DATA = [
+    ('manage_people', '*', 'Account Observer', True),
+	('manage_people', '*', 'AccountAdmin', True),
```

Comment:
Do we have variations on `AccountAdmin` that need to be used (e.g. for QA)? or perhaps we can fix QA to match the string in prod, if we haven't already?